db/migrate/20160610151323_create_posts.rb

```diff
@@ -0,0 +1,9 @@
+class CreatePosts < ActiveRecord::Migration
+  def change
+    create_table :posts do |t|
+      t.string :title
```

Comment:
By record, I'm referring to an instance of a `Post` within Rails. Let me know if you want more clarification.